internal/models/codesystem.xo.go

```diff
@@ -158,13 +158,13 @@ func CodeSystemByID(ctx context.Context, db DB, id string) (*CodeSystem, error)
 		`WHERE id = $1`
 	// run
 	logf(sqlstr, id)
-	cs := CodeSystem{
+	view := CodeSystem{
 		_exists: true,
 	}
-	if err := db.QueryRowContext(ctx, sqlstr, id).Scan(&cs.Oid, &cs.ID, &cs.Name, &cs.Definitiontext, &cs.Status, &cs.Version, &cs.Versiondescription, &cs.Assigningauthorityversionname, &cs.Distributionsourceversionname, &cs.Distributionsourceid, &cs.Assigningauthorityid, &cs.Codesystemcode, &cs.Sourceurl, &cs.Hl70396identifier, &cs.Legacyflag, &cs.Statusdate, &cs.Acquireddate, &cs.Effectivedate, &cs.Expirydate, &cs.Assigningauthorityreleasedate, &cs.Distributionsourcereleasedate, &cs.Sdocreatedate, &cs.Lastrevisiondate); err != nil {
+	if err := db.QueryRowContext(ctx, sqlstr, id).Scan(&view.Oid, &view.ID, &view.Name, &view.Definitiontext, &view.Status, &view.Version, &view.Versiondescription, &view.Assigningauthorityversionname, &view.Distributionsourceversionname, &view.Distributionsourceid, &view.Assigningauthorityid, &view.Codesystemcode, &view.Sourceurl, &view.Hl70396identifier, &view.Legacyflag, &view.Statusdate, &view.Acquireddate, &view.Effectivedate, &view.Expirydate, &view.Assigningauthorityreleasedate, &view.Distributionsourcereleasedate, &view.Sdocreatedate, &view.Lastrevisiondate); err != nil {
 		return nil, logerror(err)
 	}
-	return &cs, nil
+	return &view, nil
```

Comment:
How did all of this `cs` vs `view` code change with this PR? These files are mostly auto-generated, yes?

Reply:
OOPS, good catch. This probably happened because I still don't know how to do find and replace properly in my Neovim setup 😅